test

```diff
@@ -11,7 +11,7 @@ set -e
 
 source ./build
 
-TESTABLE="plugins/ipam/dhcp plugins/main/loopback pkg/invoke pkg/ns pkg/skel pkg/types"
+TESTABLE="plugins/ipam/dhcp plugins/main/loopback plugins/ipam/host-local pkg/invoke pkg/ns pkg/skel pkg/types"
 FORMATTABLE="$TESTABLE libcni pkg/ip pkg/ns pkg/types pkg/ipam plugins/ipam/host-local plugins/main/bridge plugins/meta/flannel plugins/meta/tuning"
```

Comment:
While you're at it, would you mind removing all the package names from `FORMATTABLE` that are now present in `TESTABLE`?